RF24Gateway.h

```diff
@@ -235,8 +260,8 @@ class RF24Gateway
     std::queue<msgStruct> rxQueue;
     std::queue<msgStruct> txQueue;
 
-    void printPayload(std::string buffer, std::string debugMsg = "");
-    void printPayload(char* buffer, int nread, std::string debugMsg = "");
+    // void printPayload(std::string buffer, std::string debugMsg = "");
+    // void printPayload(char* buffer, int nread, std::string debugMsg = "");
```

Comment:
These declarations are commented out because no definitions exist (and haven't existed). This is a reference to my other comment.